src/quo2/components/wallet/account_overview/component_spec.cljs

```diff
@@ -0,0 +1,36 @@
+(ns quo2.components.wallet.account-overview.component-spec
+  (:require [quo2.components.wallet.account-overview.view :as account-overview]
+            [test-helpers.component :as h]))
+
+(h/describe "Account overview test"
+  (h/test "renders correct account name"
+    (h/render [account-overview/account-overview
+               {:account-name "Diamond Hand"}])
+    (-> (h/get-by-text "Diamond Hand")
+        h/is-truthy))
+
+  (h/test "renders correct account value"
+    (h/render [account-overview/account-overview
+               {:current-value "20.0$"}])
+    (-> (h/get-by-text "20.0$")
+        h/is-truthy))
+
+  (h/test "renders correct account value from date to date"
```

Comment:
It would be nice to have each variant of time-frame etc covered with tests

Reply:
Will do